### ISSUE TITLE
Add feature to auto-update game scripts.

### ISSUE THREAD

**Issue by MavropaliasG** [lutris/lutris#958]
Hello, this is an idea from [a discussion we had on reddit](https://www.reddit.com/r/linux_gaming/comments/8v8tar/lutris_blizzard_games_update_2_july_by_dox/e1lism1/).

It would be great (and I think easy to implement?) to have a feature that checks for updated scripts upon opening lutris and informing you if you want to update them (to improve performance, or fix bugs).

**Comment by retiform** [comment]
This is currently the biggest issue with lutris.  I install a game like overwatch, and then find that the configuration needed/recommended has changed and have to go play a game of reading the new config file for changes.

**Comment by strycore** [comment]
yeah, I marked it for 0.7 but it might come earlier, I'm already thinking about ways to implement that.

**Comment by werymanen** [comment]
Ok sooo, what's up now?

**Comment by maker2413** [comment]
Any update on this?

**Comment by Anti-Apple4life** [comment]
Any update on 2024
